ivy/functional/frontends/numpy/linear_algebra/matrix_and_vector_products.py

```diff
@@ -4,3 +4,13 @@
 
 def outer(a, b, out=None):
     return ivy.outer(a, b, out=out)
+
+
+#inner
+
+# global
+import ivy
+
+
+def inner(a, b, out=None):
+    return ivy.inner(a, b, out=out)
```

Comment:
This is calling a backend implementation which didn't exist- I have just added it. Please merge with master to get this update.